src/main/java/org/opentripplanner/routing/api/request/refactor/request/TransitRequest.java

```diff
@@ -0,0 +1,62 @@
+package org.opentripplanner.routing.api.request.refactor.request;
+
+import java.util.List;
+import java.util.Set;
+import org.opentripplanner.routing.api.request.DebugRaptor;
+import org.opentripplanner.routing.core.RouteMatcher;
+import org.opentripplanner.transit.model.basic.MainAndSubMode;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+
+public class TransitRequest {
+  // TODO: 2022-08-17 Should be a set?
+  private List<MainAndSubMode> modes = MainAndSubMode.all();
```

Comment:
Probably vise to keep the existing collection type to avoid problems, we can resolve the todo later.